FAIL
Kernel panic in run: PANIC at ../../threads/interrupt.c:361 in intr_handler(): Unexpected interrupt
Call stack: 0x8004213866 0x8004208c61 0x8004209013 0x800420a2e8 0x8004216999 0x8004216a46 0x8004210d69 0x8004216d61 0x8004217de8 0x8004216ca2 0x8004206636 0x8004206783 0x8004206120
Translation of call stack:
0x0000008004213866: debug_panic (lib/kernel/debug.c:32)
0x0000008004208c61: intr_handler (threads/interrupt.c:365)
0x0000008004209013: intr_entry (threads/intr-stubs.o:?)
0x000000800420a2e8: lock_release (threads/synch.c:262)
0x0000008004216999: release_console (lib/kernel/console.c:103)
0x0000008004216a46: vprintf (lib/kernel/console.c:125)
0x0000008004210d69: printf (lib/stdio.c:80)
0x0000008004216d61: msg (tests/threads/tests.c:72)
0x0000008004217de8: test_priority_donate_one (tests/threads/priority-donate-one.c:42)
0x0000008004216ca2: run_test (tests/threads/tests.c:57)
0x0000008004206636: run_task (threads/init.c:252)
0x0000008004206783: run_actions (threads/init.c:297)
0x0000008004206120: main (threads/init.c:125)
